lib/engine/step/tracker.rb

```diff
@@ -83,7 +83,7 @@ def abilities(entity, **kwargs, &block)
         @game.abilities(entity, :tile_lay, **kwargs, &block)
       end
 
-      def lay_tile(action, extra_cost: 0, entity: nil, spender: nil)
+      def lay_tile(action, extra_cost: 0, entity: nil, spender: nil, ignore_abilities: nil)
```

Comment:
What is the problem you're trying to solve here?

Reply:
The special Tasmania scenario i described above.
At that point either the player or owning corp is placing the tile. I could temporarily add the ability to either with checks, I thought this was easier.

Reply:
Workarounds/hacks like this are bad in the common code. For example, if every game added their workarounds here, the code would be unreadable. It's more acceptable if the workaround is in game specific code.

I'm trying to think of an implemented game that allows the player to act on behalf of the company for a tile lay, but I came up empty. The closest I came up with was the Steamboat private in 46 that can be assigned by the owner at the beginning of the OR.

If the entity is the tasmania company, does it not work?

Reply:
found a better way of doing this, in tasmania I add a track laying ability to the owning player, and use the regular flow.